### DISCUSSION TITLE
Ruby 3.3

### DISCUSSION THREAD

**Discussion by dem** [mongodb/mongoid#5828] [General]
Ruby 3.3 was released  in the end of 2023 (2023-12-25)

Any chances to add support of it in compatibility matrix?
https://www.mongodb.com/docs/mongoid/current/reference/compatibility/#ruby-compatibility

**Comment by jamis** [comment]
Hey @dem, thanks for the heads-up. Yeah, we'll get that sorted in a release soon, we've got on it on our radar. 👍

**Reply by dem** [reply]
Thanks a million @jamis
